src/pages/landingPages/donate/components/titleSection/titleSection.js

```diff
@@ -29,10 +38,11 @@ const TitleSection = () => {
 								"Women of color and non-binary, talking to each other in a group"
 							}
 							className="women-of-color"
+							loading="lazy"
 						/>
 					</div>
 				</div>
-				<div className="icon-container">
+				<div className="icon-container-1">
```

Comment:
I can't see the rocket ship either, is it supposed to show?

Reply:
For screens smaller than 1600px, the star and rocket ship images are hidden.

Reply:
Got it, thanks for explaining that!